airbyte-integrations/connectors/source-faker/source_faker/source.py

```diff
@@ -0,0 +1,146 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+
+import json
+import os
+from datetime import datetime
+from typing import Dict, Generator
+
+from airbyte_cdk.logger import AirbyteLogger
+from airbyte_cdk.models import (
+    AirbyteCatalog,
+    AirbyteConnectionStatus,
+    AirbyteMessage,
+    AirbyteRecordMessage,
+    AirbyteStateMessage,
+    AirbyteStream,
+    ConfiguredAirbyteCatalog,
+    Status,
+    SyncMode,
+    Type,
+)
+from airbyte_cdk.sources import Source
+from faker import Faker
+
+
+class SourceFaker(Source):
+    def check(self, logger: AirbyteLogger, config: json) -> AirbyteConnectionStatus:
```

Comment:
type hint on `config` across all the funcs could be more pythonic e.g. Dict or Mapping[X, Y]

I think config should be Mapping[str, Any]

Reply:
+1 for stronger types! It looks like `Mapping[]` isn't a thing though, so I went with `config: Dict[str, any]`

<img width="509" alt="Screen Shot 2022-04-14 at 9 27 20 AM" src="https://user-images.githubusercontent.com/303226/163432709-59f47360-4219-4254-9eee-e51164df418f.png">

Reply:
@evantahler I think Dict is 100% fine, but for future reference, `Mapping` can be imported from `typing`